### ISSUE TITLE
download speed act as downloded data..

### ISSUE THREAD

**Issue by sanjeet123456789** [transmission/transmission#1351]
I just change my local date to one month past and this issue arises..with this change downloading speed label acting like as downloaded data...

![86548348-b05b7600-bf59-11ea-8080-ffac41a435f0](https://user-images.githubusercontent.com/33481706/87267927-a9e37600-c4e6-11ea-8b55-6834022bba07.png)

![index](https://user-images.githubusercontent.com/33481706/87267877-815b7c00-c4e6-11ea-82db-417686c0db3a.jpeg)

**Comment by mikedld** [comment]
@sanjeet123456789, to be clear, did you adjust the time while GTK client was running, or before starting it?

**Comment by ckerr** [comment]
Closing due to lack of response. If this is still an issue in `4.0.0-beta.1` or newer, please file a new issue.
